scripts/post-build.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+if [ "$CODEBUILD_BUILD_SUCCEEDING" = "1" ]; then
+  echo "Build succeeded for source version $CODEBUILD_SOURCE_VERSION"
+else
+  echo "TODO rollback if deployed"
```

Comment:
If we deployed the backend and frontend, but our e2e tests fail, we have to manually re-deploy the old code for now. In the future hopefully the serverless framework will make this easier.